### PR TITLE
in diagram, delete key will delete the entity (#64 #54)

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
#64 Delete key remove the entity
adding a context menu to remove from view when possible (ie in a perspective view/ non root)
#54 adding a context menu to duplicate an entity

### CODE REVIEW

File: package.nls.json [hunk 1]
```diff
@@ -9,6 +9,8 @@
     "taipy.config.commands.taipy.config.file.create": "Taipy: Create New Config file",
     "taipy.config.commands.taipy.details.showLink": "Taipy: Show Source",
     "taipy.config.commands.taipy.explorer.file.setMainModule": "Taipy: Set as main module",
+    "taipy.config.commands.taipy.perspective.duplicateNode": "Taipy: Duplicate entity",
```
Comment (FabienLelaquais) [inline package.nls.json]: I'm...not sure you're talking Entities here. Configuration elements, right?

**Review by FabienLelaquais — APPROVED** [review]
Honestly? If it works... it works.
Well done!

File: package.nls.json (hunk 1 above)
Comment (FredLL-Avaiga) [reply package.nls.json]: indeed
"Duplicate Configuration element" ?

Comment (FredLL-Avaiga) [reply package.nls.json]: or "Duplicate element" @FabienLelaquais ?

Comment (FredLL-Avaiga) [reply package.nls.json]: "Taipy: Duplicate" is enough

**Pull request merged**